pwndbg/gdblib/file.py

```diff
@@ -35,22 +35,32 @@ def remote_files_dir():
     return _remote_files_dir
 
 
-def get_file(path: str) -> str:
+def get_proc_exe_file() -> str:
+    """
+    Returns the local path to the debugged file name.
+    """
+    return get_file(pwndbg.gdblib.proc.exe, try_local_path=True)
+
+
+def get_file(path: str, try_local_path: bool = False) -> str:
     """
     Downloads the specified file from the system where the current process is
     being debugged.
 
     If the `path` is prefixed with "target:" the prefix is stripped
     (to support remote target paths properly).
 
+    If the `try_local_path` is set to `True` and the `path` exists locally and "target:" prefix is not present, it will return the local path instead of downloading the file.
```

Comment:
When do we *not* want this behavior? Maybe we should either default it to True, or just remove the option and always do this behavior (just thinking out loud, haven't dug into whether this is a bad idea or not).

Reply:
I didn't set it to default before we do something like this in many places:
https://github.com/pwndbg/pwndbg/blob/9d22acc1d769d02aae88f9b343d2043437fca6f2/pwndbg/gdblib/net.py#L13

In the above cases, seems like we can't be sure if we need to use the local path or the remote path.